doc/config.md

```diff
@@ -448,6 +448,12 @@ HBA configuration file to use when `auth_type` is `hba`.
 
 Default: not set
 
+### auth_pgident_file
```

Comment:
Let's call this auth_ident_file to use teh same convention as hba_file (that's not called auth_pghba_file either).
```suggestion
### auth_ident_file
```